changelogs/unreleased/65419.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "Move : Fix NPE when we click on daybook button with a moveLine that contains no invoiceTerm"
```

Comment:
Please don't use technical terms like `NPE` or `invoiceTerm` in changelog (`invoice term` is fine though)